web_development_101/project_html_css.md

```diff
@@ -1587,6 +1587,8 @@ Submit a pull request with a link to your solution in this [file](https://github
 * [Emil Dimitrov's Solution (Easy)](https://github.com/imemdm/google-homepage) - [View in Browser](https://imemdm.github.io/google-homepage/)
 * [Emil Dimitrov's Solution (Difficult)](https://github.com/imemdm/google-homepage) - [View in Browser](https://imemdm.github.io/google-homepage/result.html)
 * [Waris's solution (Easy)](https://github.com/warisz/google-homepage)  - [View in Browser](https://warisz.github.io/google-homepage/)
+* [Ajay Sarwan's solution](https://github.com/AjaySarwan/google-homepage) [result](https://github.com/AjaySarwan/google-homepage/blob/master/index.html)
```

Comment:
The result here is a 404 error. If you click settings you can enable GitHub pages on the repository and allow the page to be viewable by other students. If you would like help with this feel free to ask in the gitter chat rooms.